<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RendererApplicationClass</class>
 <widget class="QMainWindow" name="RendererApplicationClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RendererApplication</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>着色模式</string>
    </property>
    <widget class="QMenu" name="menuBlinn_Phong">
     <property name="title">
      <string>Blinn Phong</string>
     </property>
     <addaction name="actionflat_shading"/>
     <addaction name="actionGourand_shading"/>
     <addaction name="actionPhong_shading"/>
    </widget>
    <addaction name="menuBlinn_Phong"/>
    <addaction name="actionMesh"/>
    <addaction name="actionPBR"/>
   </widget>
   <widget class="QMenu" name="menuFunction">
    <property name="title">
     <string>可选功能</string>
    </property>
    <addaction name="actionClipping"/>
    <addaction name="bilinear"/>
    <addaction name="actionShadow_mapping"/>
   </widget>
   <widget class="QMenu" name="menuAntiAliasing">
    <property name="title">
     <string>抗锯齿模式</string>
    </property>
    <addaction name="actionMSAA"/>
    <addaction name="actionSSAA"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen_new_model"/>
    <addaction name="actionadd_model"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuMode"/>
   <addaction name="menuFunction"/>
   <addaction name="menuAntiAliasing"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionMesh">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>mesh</string>
   </property>
  </action>
  <action name="actionPBR">
   <property name="text">
    <string>PBR()</string>
   </property>
  </action>
  <action name="actionflat_shading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Flat shading</string>
   </property>
  </action>
  <action name="actionGourand_shading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gourand shading</string>
   </property>
  </action>
  <action name="actionPhong_shading">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Phong shading</string>
   </property>
  </action>
  <action name="actionClipping">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>clipping</string>
   </property>
  </action>
  <action name="bilinear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>bilinear(texture)</string>
   </property>
  </action>
  <action name="actionShadow_mapping">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>shadow mapping</string>
   </property>
  </action>
  <action name="actionMSAA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>MSAA</string>
   </property>
  </action>
  <action name="actionSSAA">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SSAA</string>
   </property>
  </action>
  <action name="actionadd_model">
   <property name="text">
    <string>add model</string>
   </property>
  </action>
  <action name="actionopen_new_model">
   <property name="text">
    <string>open new model</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="RendererApplication.qrc"/>
 </resources>
 <connections/>
</ui>
